.travis.yml

```diff
@@ -0,0 +1,25 @@
+sudo: required
```

Comment:
i don't think we need `sudo: required`? Note [the benefits of using `sudo: false`](https://docs.travis-ci.com/user/ci-environment/#Virtualization-environments)